axelor-production/src/main/java/com/axelor/apps/production/service/operationorder/OperationOrderWorkflowServiceImpl.java

```diff
@@ -297,30 +298,34 @@ protected List<OperationOrder> getNextOrderedOperationOrders(OperationOrder oper
   @Override
   @Transactional(rollbackOn = {Exception.class})
   public void start(OperationOrder operationOrder) throws AxelorException {
-    if (operationOrder.getStatusSelect() != OperationOrderRepository.STATUS_IN_PROGRESS) {
-      operationOrder.setStatusSelect(OperationOrderRepository.STATUS_IN_PROGRESS);
-      operationOrder.setRealStartDateT(appProductionService.getTodayDateTime().toLocalDateTime());
-
-      startOperationOrderDuration(operationOrder);
-
-      if (operationOrder.getManufOrder() != null) {
-        int beforeOrAfterConfig =
-            operationOrder.getManufOrder().getProdProcess().getStockMoveRealizeOrderSelect();
-        if (beforeOrAfterConfig == ProductionConfigRepository.REALIZE_START) {
-          for (StockMove stockMove : operationOrder.getInStockMoveList()) {
-            Beans.get(ManufOrderStockMoveService.class).finishStockMove(stockMove);
-          }
-
-          StockMove newStockMove =
-              operationOrderStockMoveService._createToConsumeStockMove(
-                  operationOrder, operationOrder.getManufOrder().getCompany());
-          newStockMove.setStockMoveLineList(new ArrayList<>());
-          Beans.get(StockMoveService.class).plan(newStockMove);
-          operationOrder.addInStockMoveListItem(newStockMove);
+
+    if (operationOrder.getStatusSelect() == OperationOrderRepository.STATUS_IN_PROGRESS
+        || !canStartOperationOrder(operationOrder)) {
+      return;
+    }
+
+    operationOrder.setStatusSelect(OperationOrderRepository.STATUS_IN_PROGRESS);
+    operationOrder.setRealStartDateT(appProductionService.getTodayDateTime().toLocalDateTime());
+
+    startOperationOrderDuration(operationOrder);
+
+    if (operationOrder.getManufOrder() != null) {
+      int beforeOrAfterConfig =
+          operationOrder.getManufOrder().getProdProcess().getStockMoveRealizeOrderSelect();
+      if (beforeOrAfterConfig == ProductionConfigRepository.REALIZE_START) {
+        for (StockMove stockMove : operationOrder.getInStockMoveList()) {
+          Beans.get(ManufOrderStockMoveService.class).finishStockMove(stockMove);
```

Comment:
No injection with `Beans.get` in services, only by constructor therefore please replace occurences of Beans.get (only thoses that you added)

Reply:
this issue will be fixed in a future PR for older versions as well, as this could cause performance issues.